python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/community-lists.j2

```diff
@@ -4,7 +4,7 @@
  that can be found in the LICENSE file.
 #}
 {# eos - community lists #}
-{% if community_lists is arista.avd.defined %}
+{% if community_lists is arista.avd.defined and not ip_community_lists is arista.avd.defined %}
```

Comment:
Why are you introducing the precedence here?
If we keep this, we need an entry in the porting guide to tell people to only use the new one as if they were using both this would change their config.

Reply:
Removed the prefence.